routes/api/contacts.js

```diff
@@ -1,25 +1,75 @@
-const express = require('express')
+const express = require("express");
+const Joi = require("joi");
+const {
+  listContacts,
+  getContactById,
+  addContact,
+  removeContact,
+  updateContact,
+} = require("../../models/contacts");
 
-const router = express.Router()
+const contactSchema = Joi.object({
+  name: Joi.string().min(3).max(30).required(),
+  email: Joi.string().email().required(),
+  phone: Joi.string().min(7).max(15).required(),
+});
 
-router.get('/', async (req, res, next) => {
-  res.json({ message: 'template message' })
-})
+const router = express.Router();
 
-router.get('/:contactId', async (req, res, next) => {
-  res.json({ message: 'template message' })
-})
+router.get("/", async (req, res, next) => {
+  try {
+    const contacts = await listContacts();
+    res.json(contacts);
+  } catch (error) {
+    next(error);
+  }
+});
 
-router.post('/', async (req, res, next) => {
-  res.json({ message: 'template message' })
-})
+router.get("/:contactId", async (req, res, next) => {
+  try {
+    const contact = await getContactById(req.params.contactId);
+    if (!contact) {
+      return res.status(404).json({ message: "Not found" });
+    }
+    res.json(contact);
+  } catch (error) {
```

Comment:
warto consologować błędy wiesz, bo kiedy coś nie działa to za każdym razem trzeba będzie pisać console.log tutaj i tam gdzie masz catch :-) Na początek to wystarczy a przyśpiesza pracę nad projektem. ;)